splitio/engine/strategies/strategy_optimized_mode.py

```diff
@@ -0,0 +1,32 @@
+from splitio.engine.strategies.base_strategy import BaseStrategy
+from splitio.engine.strategies import Observer, Counter, truncate_impressions_time
+from splitio import util
+
+class StrategyOptimizedMode(BaseStrategy):
+    """Optimized mode strategy."""
+
+    def __init__(self, counter=None, observer=None, standalone=True):
+        """
+        Construct a strategy instance for optimized mode.
+
+        """
+        self._standalone = standalone
+        self._counter = counter
+        self._observer =  observer
```

Comment:
observer can be instantiated here and not received as parameter

Reply:
removed